//= error

fun main(): Unit {
    std::call("unknown fct");
}